fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -95,7 +99,7 @@ def from_pymongo(self, doc: dict):
         self.parent_id = doc.get("parent_id", None)
 
         # internal fields
-        self.id = doc["_id"]
+        self.id = doc.get("_id", doc.get("id"))
```

Comment:
since _id is usually ObjectId and id is a string, are we saying self.id is either?

Reply:
yeah uhh idk. We should cast to `str` I guess